test/unit/dom/dom.js

```diff
@@ -200,9 +588,33 @@ describe('skatejs-named-slots dom', function () {
 
         host.innerHTML = processingInstruction;
         expect([processingInstruction, processingInstructionsAfterInnerHtml].indexOf(host.innerHTML)).to.be.above(-1);
+        expect(host.childNodes.length).to.equal(1);
+        expect(slot.getAssignedNodes().length).to.equal(1);
```

Comment:
This support was always there, there were never any tests to prove it. We do not distinct between nodes when we are doing node assigning.